AutomateQuizInput/Program.cs

```diff
@@ -18,6 +18,8 @@ static void Main(string[] args)
             var allLines = Helper.ReadDocument(path);
             var separatedQuizzes = Helper.SeparateQuizzes(allLines);
             List<Quiz> completeQuizzes = new List<Quiz>();
+            List<string> list = new List<string>();
```

Comment:
I don't see the variables on these two lines being used. Can we remove them?

Reply:
Yes they will  be gone soon.